<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>0</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="inputLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="DrawButton">
       <property name="text">
        <string>Отрисовать график</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="text">
        <string>Очистить экран</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="iterationLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Число разбиений (осн. сетка):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IterationCount">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="iterationLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Число разбиений (контр. сетка):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IterationCount_c">
         <property name="text">
          <string>999</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="chartLayout"/>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="aux_info">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="taskSelectionLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="testTask">
       <property name="text">
        <string>Тестовая задача</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mainTask">
       <property name="text">
        <string>Основная задача</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="oscillTask">
       <property name="text">
        <string>Осциллирующая задача</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>280</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="funcSelectionLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="funct">
       <property name="text">
        <string>Функция</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diriv">
       <property name="text">
        <string>Первая производная</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diriv2">
       <property name="text">
        <string>Вторая производная</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_sub">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="funcSelectionLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="naturalGU">
       <property name="text">
        <string>Естественные ГУ</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="accurateGU">
       <property name="text">
        <string>Точные ГУ</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
